src/components/listview/listview.css

```diff
@@ -100,7 +100,7 @@
     min-width: 2.78em;
     min-height: 2.78em;
     background-repeat: no-repeat;
-    background-size: contain;
+    background-size: cover;
```

Comment:
just curious - what this change does?

Reply:
In the list section for a given serie (ex. serie 1) the cover for each episode was not right, making the seen mark not being aligned with it. That was caused because of the improper disposition of the cover image of the episode. This solves that issue.